site/docs/v1/tech/apis/_key-generate-post-request-body.adoc

```diff
@@ -14,8 +14,8 @@ The algorithm used to generate the Key. The following values represent algorithm
 * `HS384` - HMAC using SHA-384 hash algorithm
 * `HS512` - HMAC using SHA-512 hash algorithm
 
-[field]#key.issuer# [type]#[String]# [optional]#Optional#::
-The issuer of the RSA or EC certificate. If omitted,this value will default to the issuer defined on the tenant. For HMAC keys, this field does not apply and will be ignored if specified.
+[field]#key.issuer# [type]#[String]# [optional]#Optional# [default]#defaults to `tenant.issuer` for default tenant#::
+The issuer of the RSA or EC certificate. If omitted, this value will default to the value of [field]#tenant.issuer# on the default tenant. For HMAC keys, this field does not apply and will be ignored if specified.
```

Comment:
Usually when we have a default that requires some explanation, we do this `[default]#defaults to **[see description]**#`. 

Look for usages like that for examples. Using that style, this would be:

```
[field]#key.issuer# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
The issuer of the RSA or EC certificate. 
+
If omitted, this value will default to the value of [field]#tenant.issuer# on the default tenant.  For HMAC keys, this field does not apply and will be ignored if specified, and no default value will be set.
```

Reply:
Ah ok -  Missed this.  I did search for examples and spot-check this on local, and was concerned that it looked a bit "clunky" initially 🙂  This is much better.